packages/apps/types/src/schema/sketch.ts

```diff
@@ -2,9 +2,13 @@
 // Copyright 2024 DXOS.org
 //
 
-import { Expando, ref, S, TypedObject } from '@dxos/echo-schema';
+import { ref, S, TypedObject } from '@dxos/echo-schema';
 
-export class SketchType extends TypedObject({ typename: 'braneframe.Sketch', version: '0.1.0' })({
-  title: S.optional(S.String),
-  data: ref(Expando),
+export class CanvasType extends TypedObject({ typename: 'dxos.org/type/Canvas', version: '0.1.0' })({
```

Comment:
@dmaretskyi should we reuse the TextType here? 
I guess TLDraw treats this as a append-only text blob?

Reply:
afaik tldraw format is a structured json-like object, not text